src/pages/index.tsx

```diff
@@ -26,59 +28,65 @@ const Home: React.FC = () => {
   ];
 
   return (
-    <div className={styles.container}>
-      <Head>
-        <title>Home Page</title>
-        <link rel="stylesheet" href="https://starter-kit-demo.herokuapp.com/styles/index.css" />
-      </Head>
-      <div>
-        <select size={11} value={selectedOption} onChange={(e) => setSelectedOption(e.target.value)}>
-          <option value="1">Button</option>
-          <option value="2">Neutral Button</option>
-          <option value="3">Brand Button</option>
-          <option value="4">Card</option>
-          <option value="5">Collapsed Card</option>
-          <option value="6">Loading Card</option>
-          <option value="7">Accordion</option>
-          <option value="8">Toast</option>
-          <option value="9">Success Toast</option>
-          <option value="10">Warning Toast</option>
-          <option value="11">Error Toast</option>
-        </select>
-      </div>
+    <ToastProvider>
```

Comment:
za duży diff, toast powinien być tylko do story z toastem; przykład: jeśli w ToastProvider będzie jakiś error renderowania, to wszystko się nam zepsuje, tak więc ToastProvider z większym dospecyfikowaniem istnienia, najlepiej "za ifem"

Reply:
za ifem oznacza, że prawdopodobniej będzie wiel razy musiał być powtórzony, bo mamy wiele ToastStory :) być może ToastProvider wrzucić do ToastStory :)